.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+<!--
+Thank you for contributing to `too-many-hooks`! You must fill out the information below before we can review this pull request. By explaining why you're making a change (or linking to an issue) and what changes you've made, we will be best able to review your PR.
+-->
+
+### Why:
+
+Closes ISSUE
+
+<!-- If there's an existing issue for your change, please replace ISSUE above with a link to the issue.
```

Comment:
We should include an example showing that they can just type the number (e.g. Closes #29)